03-using-containers.qmd

```diff
@@ -58,12 +58,12 @@ Which means: If your computer has files, software, or anything else. Your contai
 ottrpal::include_slide("https://docs.google.com/presentation/d/1T5Lfei2UVou9b0qaUCrWXmkcIwAao-UcN4pHMPEE4CY/edit#slide=id.g30a4ed49e59_0_713")
 ```
 
-We will encounter both an error of when we don't have a file we need AND an error of when we don't have a software package we need. We will guide you through what to do when these expected errors occur.
+We will encounter both an error when we don't have a file we need AND an error when we don't have a software package we need. We will guide you through what to do when these expected errors occur.
 
 #### Step 1: Docker pull
 
 We need to get the Docker **image** we want to use. We do this by "pulling" it.
-Pulling is a term used in GitHub terminology too. It just means we're taking something from the internet and getting a copy locally (on our computer).
+Pulling is a term used in GitHub terminology too. It just means we're taking something from the internet and getting a copy locally (on our computer). Pulling is basically like downloading.
```

Comment:
Trying to provide an example for people new to containers but comfortable with computers